docs/integrations_api.yml

```diff
@@ -0,0 +1,134 @@
+swagger: '2.0'
+info:
+  version: '0.1'
+  title: User administration and authentication
+  description: |
+    An API for user administration and user authentication handling. Intended for use by the web GUI.
+
+basePath: '/api/integrations/0.1/users'
```

Comment:
can't come up with a different human-friendly name. `useradmin`? `useradm` is too short. `user_management`?